python/nav/portadmin/snmp/cisco.py

```diff
@@ -300,5 +315,88 @@ def get_dot1x_enabled_interfaces(self):
             for oid, state in self._bulkwalk(self.dot1xPortAuth)
         }
 
+    def get_poe_state_options(self) -> Sequence[PoeState]:
+        """Returns the available options for enabling/disabling PoE on this netbox"""
+        options_list = [
+            self.POE_AUTO,
+            self.POE_STATIC,
+            self.POE_LIMIT,
+            self.POE_DISABLE,
+        ]
+        return options_list
```

Comment:
Unless this list is expected to be dynamically generated at some point, I suggest this method should just return a global constant, rather than construct the same list each time it is called.